plutus-pab/app/Command.hs

```diff
@@ -17,7 +17,7 @@ import           Wallet.Types                            (ContractInstanceId)
 
 -- | A command for which a config.yaml file is required
 data ConfigCommand =
-    MockNode -- ^ Run the mock node service
+    MockNode Bool -- ^ Run the mock node service without starting the server
```

Comment:
Should be a datatype, something like `MockServerMode = WithMockServer | WithoutMockServer`